src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CommandResolverWithVectorizerTest.java

```diff
@@ -421,183 +415,6 @@ public void findOneAndDelete() throws Exception {
               });
     }
 
-    @Test
```

Comment:
We do not vectorize updateCommand in resolver level anymore.
Which will invalid these tests.